build.gradle

```diff
@@ -103,6 +107,30 @@ javadoc {
     }
 }
 
+
+
+if (project.hasProperty('githubToken')) {
```

Comment:
How do I create this token value? `-PgithubToken={$javadocGithubToken}` in [buildViaTravis](https://github.com/ReactiveX/RxJava/blob/2.x/gradle/buildViaTravis.sh#L21)?